app/models/est.rb

```diff
@@ -0,0 +1,35 @@
+class Est
+  include Indexable
+
+  self.settings = {
+    index: {
+      analysis: {
+        analyzer:
+            { custom_analyzer:        {
+              tokenizer: 'standard',
+              filter:    %w(standard asciifolding lowercase snowball) },
+            },
+      },
+    },
+  }.freeze
+
+  self.mappings = {
+    name.typeize => {
+      _timestamp: {
+        enabled: true,
+        store:   true,
+      },
+      dynamic:    'false',
+      properties: {
+        name_chinese:      { type: 'string', analyzer: 'custom_analyzer' },
```

Comment:
As mentioned in my [earlier comment](https://github.com/GovWizely/webservices/pull/161/files#r32501928), yes please analyze all these by creating an analyzer for each language. snowball isn't suitable for Chinese though, but maybe just the standard word-based filter is good enough.

It will be worth adding specs to show some of these stemming/folding behavior in action :wink: